.dockerignore

```diff
@@ -39,7 +39,7 @@ contracts/.git
 !etc/multivm_bootloaders
 !cargo
 !bellman-cuda
-!prover/vk_setup_data_generator_server_fri/data/
+!prover/crates/bin/vk_setup_data_generator_server_fri/data/
```

Comment:
This was and is suboptimal, but I would prefer to tackle it separately.
This folder should be somewhere near the root and outside of any crate.